opacclient/opacapp/src/main/AndroidManifest.xml

```diff
@@ -136,8 +138,11 @@
             android:finishOnTaskLaunch="true"
             android:launchMode="singleInstance"
             android:theme="@style/Theme.AppCompat.Light.Dialog" />
+        <activity
+            android:name="de.geeksfactory.opacclient.frontend.SnoozeDatePickerActivity"
+            android:theme="@style/Theme.Opacapp.Transparent" />
 
-        <service android:name="de.geeksfactory.opacclient.reminder.ReminderCheckService" >
+        <service android:name=".reminder.SyncAccountService">
```

Comment:
I guess we always use full-qualified class paths here because it makes life simpler for the Plus edition. I'm unsure whether this still holds, but we should either test it or just ontinue using a full class path.